### PR TITLE
v2v: add checkpoint for glance deprecate on RHEL9

### PR DESCRIPTION
**Pull request by vwu-vera**
Check bz #1977539.

Signed-off-by: vwu-vera <vwu@redhat.com>

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.machine_readable.input_mode.none.output_mode.none: PASS (2.61 s)

File: v2v/tests/src/v2v_options.py [hunk 1]
```diff
@@ -559,10 +561,12 @@ def check_alloc():
                     # output regardless of the orders and new items.
                     with open(params['example_file']) as f:
                         for line in f:
+                            strippedLine = line.strip()
```
Comment (xiaodwan) [inline v2v/tests/src/v2v_options.py:564]: this style 'strippedLine'  does not conform to the code style of v2v in tp-libvirt.
It's better to use '_' to concatenate 'a variant name, such as 'stripped_line'.

**Comment by vwu-vera** [comment]
@xiaodwan Modified the 'strippedLine' to  'stripped_line' as suggested. 
Please help to review the latest one. Thanks.

**Review by xiaodwan — CHANGES_REQUESTED** [review]

**Comment by vwu-vera** [comment]
Modified the script according to the comment. And rerun passed:  (1/1) type_specific.io-github-autotest-libvirt.v2v_options.positive_test.machine_readable.input_mode.none.output_mode.none: PASS (3.28 s)

@xiaodwan Please help to review the latest commit. Thanks.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**